client/pages/create-profile/index.tsx

```diff
@@ -32,6 +32,7 @@ import {
   ButtonGreen,
   ButtonOrange,
 } from '../../styles/ui-components/Chakra-Button';
+import AddAvatar from './add-avatar';
```

Comment:
I think you're trying to import a Next.js page file and not a component. Next.js routing works a bit differently. https://nextjs.org/docs/routing/introduction

Reply:
It's a component but I might have named it or stored it in a way that isn't correct in not sure, but it's meant to be a component.

Reply:
My bad on this one, @angeljgomezc explained to me briefly how i messed up the on where i created this component, fixing this right now